src/components/v5/common/MemberSignature/MemberSignature.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { MemberSignatureProps } from './types';
+
+const displayName = 'v5.common.MemberSignature';
+
+const MemberSignature: FC<MemberSignatureProps> = ({ user, isSigned }) => {
+  const { formatMessage } = useIntl();
+
+  const { name, profile } = user;
+  const { displayName: userDisplayName } = profile || {};
+
+  return (
+    <div className="flex items-center justify-between">
```

Comment:
It would be also good to add a gap here in case the user name is long or there won't be enough space to display everything